jupyterlab/labbox_ephys_widgets_jp/src/extensions/extensionInterface.ts

```diff
@@ -10,7 +12,7 @@ export interface Sorting {
 
     sortingInfo?: SortingInfo
     externalUnitMetrics?: ExternalSortingUnitMetric[]
-    unitCuration?: {[key: string]: {labels: Label[]}}
+    curation?: SortingCuration
```

Comment:
I reworked the curation state/dispatch system in order to achieve the following.

We do not want sorting plugin views to depend on the global redux state. In particular we do not want the CurationSortingView to interact with the global redux state. So, the following props are now passed into each SortingView:

sorting: {..., curation}
curationDispatch

this curation/dispatch pair is specific to this particular sorting. Depending on the context (jupyter widget or labbox GUI) this will either hook into the python-defined curation context, or the global redux state.